single-line-diagram-core/src/main/java/com/powsybl/sld/model/Fictitious3WTNode.java

```diff
@@ -17,20 +18,43 @@
  */
 public class Fictitious3WTNode extends FictitiousNode {
 
-    private final String transformerId;
+    private final Map<Feeder3WTNode.Side, String> idsLegs;
```

Comment:
Isn't redundant with information from VoltageLevelGraph?
A node knows its (node, edge) neighbors, and should be possible to get it by just looking at neighbors.
Also maybe Fictitious3WTNode should not extend FictitiousNode and should have a more specific design
   - only 3 edges possible (so edge1, edge2, edge3 attributes and not a list)
   - these 3 edges are kind of ordered (leg 1, 2, 3)
   - it would allow also to refactor TwtEdge to be more "type safe": one side with a  Fictitious3WTNode and the other with a FeederNode

Reply:
Fictitious3WTNode is created only when we display a voltage level diagram (and not in the case of a substation or zone diagram), so we need to store in that kind of node the ids and nominal voltage of the other voltage levels ...